src/components/IncidentDetails/DefinitionOfDone/EvidenceView/index.tsx

```diff
@@ -151,11 +151,7 @@ function ConfigEvidence({
 
   return (
     <div className={clsx("overflow-hidden p-2", className)} {...rest}>
-      <Icon
-        name={config.external_type}
-        secondary={config.config_type}
-        size="lg"
-      />{" "}
+      <Icon name={config.external_type || config.config_type} />{" "}
```

Comment:
Both of these needs to passed through, external_type might not be null, but also not have a corresponding icon